packages/alchemy/src/signer/session/manager.ts

```diff
@@ -139,9 +149,39 @@ export class SessionManager {
     event: E,
     listener: SessionManagerEvents[E]
   ) => {
-    this.eventEmitter.on(event, listener as any);
-
-    return () => this.eventEmitter.removeListener(event, listener as any);
```

Comment:
isn't it a breaking change to remove this event emitter though?
The devs could be listening/relying to these events right?

Reply:
oh I guess the session manager is entirely isolated to be within the SDK only.